SqlInstall.ps1

```diff
@@ -0,0 +1,556 @@
+﻿function Install-SqlServer ([string]$AppVolume, [string]$DataVolume, [string]$LogVolume, [string]$TempVolume, [string]$BackupVolume, [int]$Ctp)
+{
+    <#
+    .SYNOPSIS
+
+    This function will help you to quickly install a SQL Server instance. 
+
+    .DESCRIPTION
+
+    This function will help you to quickly install a SQL Server instance. By scanning your system it will apply a number of best practices to your installation
+    The max DOP and max server memory will be set, the sa account will be disabled while the account used to install SQL Server will be added as sysadmin.
+    The number of TempDB files will be set to the number of cores with a maximum of eight.
+    The perform volume maintenance right will be granted to the SQL Server account. If you run this script in an environment where you are not allowed to do this,
+    please revert that operation by removing the right from the local security policy (secpol.msc).
+
+    The autogrowth of file will be set to 64MB, the Cost Threshold for Parallelism will be set to 40. 
+
+    Note that the dowloaded installation file must be unzipped, an ISO has to be mounted. This will not be executed from this script.
+    
+    .PARAMETER Name
+
+    $AppVolume will hold the volume letter of the application disc. If left empty, it will default to C, unless there is a drive named like App
```

Comment:
Doc content looks good! Just need to format it properly, like `.PARAMETER AppVolume`

Reply:
Formatting changed. I'll keep the content up to date with all the additions and changes.

Reply:
Excellent